### PR TITLE
Mention comments in cheatsheet

### PR DESCRIPTION
**Pull request by jochenschneider**
I think it would be nice to be able to quickly look up the comments syntax in the cheatsheet. (That's where I went to look for them.)

### CODE REVIEW

**Comment by sjakobi** [comment]
Cheers!

How about adding a bit of code to explain where these comments may appear in code?

File: docs/howtos/Cheatsheet.md [hunk 1]
```diff
@@ -302,3 +302,20 @@
...
     You can find latest Prelude of importable functions at https://prelude.dhall-lang.org/
+
+*   Comments
+
+    Dhall has single-line and block comments (with a syntax borrowed from Haskell):
+
+    ```dhall
+    -- single-line comment
+
+    --Another one, they don't need to starts with a space.
```
Comment (sjakobi) [inline docs/howtos/Cheatsheet.md]: Just a suggestion – the question of spaces after `--` is IMHO an unnecessary detail for a cheatsheet…

```suggestion
    True -- Line comments may follow code on the same line
```

Comment (jochenschneider) [reply docs/howtos/Cheatsheet.md]: Make sense, updated.

**Comment by jochenschneider** [comment]
Thanks for the quick feedback and approval!

**Pull request merged**